test/cusparse/conversions.jl

```diff
@@ -70,3 +76,75 @@ end
         end
     end
 end
+
+if CUSPARSE.version() >= v"11.3"
+    @testset "conversions" begin
```

Comment:
a testset "conversions" isn't really descriptive here -- the parent testset (derived from the filename) is already "conversions"

Reply:
What do you think of `conversions between CuSparseMatrices`?